futaba/cogs/info/alias.py

```diff
@@ -180,10 +177,7 @@ def __init__(self, bot):
             for i, file in enumerate(files, 1):
                 await ctx.send(content=f'#{i}', file=file)
 
-        await asyncio.gather(
-            send(),
-            Reactions.SUCCESS.add(ctx.message),
-        )
+        await send()
```

Comment:
`send()` is done as a local function so it can be `gather()`'d. it should just be removed and the previous code unindented.